types/bitcoinjs-lib/index.d.ts

```diff
@@ -232,172 +352,969 @@ export const opcodes: {
+            const prototype: {
+            };
+
+        }
+
+        namespace ucs2Write {
+            const prototype: {
+            };
+
+        }
+
+        namespace utf8Slice {
+            const prototype: {
+            };
+
+        }
+
+        namespace utf8Write {
+            const prototype: {
+            };
+
+        }
+
+        namespace write {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeDoubleBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeDoubleLE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeFloatBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeFloatLE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt16BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt16LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt32BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt32LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeInt8 {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeIntBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeIntLE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt16BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt16LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt32BE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt32LE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUInt8 {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUIntBE {
+            const prototype: {
+            };
+
+        }
+
+        namespace writeUIntLE {
+            const prototype: {
+            };
+
+        }
+
+    }
+
+
+}
+
+
+
+export namespace address {
+    function fromBase58Check(address: string): string;
+
+    function fromOutputScript(outputScript: Buffer, network: Network): string;
+
+    function toBase58Check(hash: Buffer, version: number, ...args: any[]): any;
+
+    function toOutputScript(address: string, network: Network): any;
+
+    namespace fromBase58Check {
+        const prototype: {
+        };
+
+    }
+
+    namespace fromOutputScript {
+        const prototype: {
+        };
+
+    }
+
+    namespace toBase58Check {
+        const prototype: {
+        };
+
+    }
+
+    namespace toOutputScript {
+        const prototype: {
+        };
+
+    }
+
+}
+
+export namespace bufferutils {
+    function pushDataSize(i: any): any;
+
+    function readPushDataInt(buffer: Buffer, offset: any): any;
```

Comment:
`offset` is `number`.
Returns `{ opcode: number, number: number, size: number }`.